packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -704,13 +708,20 @@ const tenantsRouter = (
       const ctx = fromAppContext(req.ctx);
 
       try {
-        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE]);
+        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE, M2M_ADMIN_ROLE]);
 
         const { tenantId } = req.params;
-        const tenant = await tenantService.addCertifiedAttribute(
-          { tenantId: unsafeBrandId(tenantId), tenantAttributeSeed: req.body },
-          ctx
-        );
+        const { data: tenant, metadata } =
+          await tenantService.addCertifiedAttribute(
+            {
+              tenantId: unsafeBrandId(tenantId),
+              tenantAttributeSeed: req.body,
+            },
+            ctx
+          );
+
+        setMetadataVersionHeader(res, metadata);
```

Comment:
Should be added to tenant API spec (see https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-a50f62f3a4939e0ad8d4f443bf76522090263ad0bfabf14e174d8d83475fc372R39)